src/test/java/net/datafaker/base/BusinessTest.java

```diff
@@ -5,19 +5,23 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 class BusinessTest extends BaseFakerTest<BaseFaker> {
-    @Test
-    void creditCardNumber() {
-        assertThat(faker.business().creditCardNumber()).isNotEmpty();
-    }
+	@Test
+	void creditCardNumber() {
+		assertThat(faker.business().creditCardNumber()).isNotEmpty();
+	}
 
-    @Test
-    void creditCardType() {
-        assertThat(faker.business().creditCardType()).isNotEmpty();
-    }
+	@Test
+	void creditCardType() {
+		assertThat(faker.business().creditCardType()).isNotEmpty();
+	}
 
-    @Test
-    void creditCardExpiry() {
-        assertThat(faker.business().creditCardExpiry()).isNotEmpty();
-    }
+	@Test
+	void creditCardExpiry() {
+		assertThat(faker.business().creditCardExpiry()).isNotEmpty();
+	}
```

Comment:
Why do we have these formatting changes?
Sometimes formatting changes are ok, however I would split logic changes and formatting changes in different commits. It would simplify going through git history while reading the code and trying to understand the logic

Reply:
Thank you for heads up! Will take care of it in future PRs :)

Reply:
Thanks for the change, appreciate it. I'll merge it and reformat it in a new PR, no problem!